index.js

```diff
@@ -813,8 +813,12 @@ module.exports = function({parent}) {
                 if (/^(date.*|smalldate.*)$/.test(column.type.declaration)) {
                     // set a javascript date for 'date', 'datetime', 'datetime2' 'smalldatetime' and 'time'
                     return new Date(value);
+                // } else if (column.type.declaration === 'time') {
+                //     return new Date('1970-01-01T' + value);
                 } else if (column.type.declaration === 'time') {
-                    return new Date('1970-01-01T' + value);
+                    let date = new Date('1970-01-01T' + value);
```

Comment:
If I understand this properly, it is easier to achieve the same by adding 'Z' , i.e. `new Date('1970-01-01T' + value + 'Z')`
Can you check if this works?

Reply:
That is correct! The change has been applied accordingly.